config/nightwatch.js

```diff
@@ -38,7 +37,6 @@ module.exports = {
         acceptSslCerts: true,
         webStorageEnabled: true,
         chromeOptions: {
-          binary: electron,
           args: ['--window-size=1024,768']
```

Comment:
Chrome was the default?

Reply:
Yeah, to use Electron you use the normal Chrome set up and driver, just with the Electron binary. So all we have to do is remove the binary to switch to normal Chrome.